circuitpython/lib/sphero_rvr.py

```diff
@@ -145,37 +135,39 @@ def reset_yaw(self):
 
         return
 
-    def set_streaming(self):
-        
+    def conf_streaming(self):
         SOP = 0x8d
-        FLAGS = 0x06
+        FLAGS = 0x02
         TARGET_ID = 0x02
-        SOURCE_ID = 0x0b
+        SOURCE_ID = 0x00
         DEVICE_ID = 0x18
         COMMAND_ID = 0x39
         SEQ = 0x01
         EOP = 0xD8
-        
+
         output_packet = [SOP, FLAGS, DEVICE_ID,COMMAND_ID,SEQ]
-        output_packet.extend([0x03, 0x00, 0x0A, 0x02, 0x00, 0x03, 0x01])
-    
+        output_packet.extend([0x02,0x00, 0x06, 0x02,0x00,0x01,0x00])
+
         output_packet.extend([~((sum(output_packet) - SOP) % 256) & 0x00FF,EOP])
+        uart.write(bytearray(output_packet))
+        return bytearray(output_packet)
 
     def start_streaming(self):
-    
         SOP = 0x8d
-        FLAGS = 0x06
-        TARGET_ID = 0x02
-        SOURCE_ID = 0x0b
+        FLAGS = 0x02
+        TARGET_ID = 0x01
+        SOURCE_ID = 0x00
         DEVICE_ID = 0x18
         COMMAND_ID = 0x3A
-        SEQ = 0x01
+        SEQ = 0x02
```

Comment:
Documentation can be found at https://sdk.sphero.com/docs/api_spec/general_api 

More object oriented programming style creation is seen here
https://github.com/sphero-inc/sphero-sdk-raspberrypi-python/blob/master/sphero_sdk/asyncio/client/toys/sphero_rvr_async.py